vmdb/config/initializers/fast_gettext.rb

```diff
@@ -18,6 +18,38 @@ def fix_i18n_available_locales
   I18n.available_locales += FastGettext.available_locales.grep(/_/).map { |i| i.gsub("_", "-") }
 end
 
+def find_available_locales_via_directories
```

Comment:
Can we move all these methods into a module or something?  I don't like that they all live in the main object.